swagger-spec/nodes/registrations_list.yaml

```diff
@@ -182,3 +182,64 @@ get:
               node_license:
             type: registrations
             id: esa63
+
+# /nodes/{node_id}/registrations/
+post:
+  summary: Create a registration
+  description: >-
+    Initiate a registration of the current draft registration.
```

Comment:
I don't know if I'd use the word "initiate" here.  This is really the final step in a multi-step process of creating a registration.  "Finalize the registration process." , maybe, followed by:

A registration is created from a completed draft registration (containing the supplemental registration questions), information about whether the registration will become public immediately or be embargoed, and conditionally, the datetime at which the registration will become public.  A registration is a frozen version of the project that can never be edited or deleted, but can be withdrawn. Your original project remains editable but will now have the registration linked to it.